pkg/integration/integration_test.go

```diff
@@ -108,6 +109,35 @@ func TestIntegrationTestSuite(t *testing.T) {
 		store:       store,
 		base:        handlers.NewHandlerBase(),
 	}
+	//TODO create user
+	getTestPrincipal(store, testSuite.user.euaID)
```

Comment:
- Can probably remove this `TODO`
- We should probably make it clear (either from comments or re-naming the function) that this doesn't just return a principal object, but also (more importantly, IMO) creates an account in the user table

Reply:
Agreed! Thanks Clay!